G/GibbsSeaWater/build_tarballs.jl

```diff
@@ -8,8 +8,7 @@ version = v"3.5.0"
 # Collection of sources required to build GibbsSeaWater
 # note 3.05.0-4 is too old to be cross-compiled on FreeBSD
 sources = [
-    "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip" =>
-    "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c",
+    "https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip" => "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c",
```

Comment:
You only changed indentation here?  Why?  Also, quoting from the [log](https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=20049&view=logs&j=96c13d9d-dfd2-51e5-2f76-4a0b3ce09abc&t=09532434-8b0a-5f0e-118d-a3181cde5ec8&l=30):
```
┌ Warning: Using a pair as source is deprecated, use ArchiveSource instead
└ @ BinaryBuilderBase /depot/packages/BinaryBuilderBase/QQlL4/src/Sources.jl:278
```
```suggestion
    ArchiveSource("https://github.com/TEOS-10/GSW-C/archive/d392e91cb63341f543ed1609c4eff613055ab3cb.zip", "454c46ec00468aeba048fa4f3cee095f927925ab699b1d1b3a25a838cef2d22c"),
```